CONTRIBUTING.md

```diff
@@ -153,6 +153,23 @@ rather than conform to specific API names or argument patterns in the spec.
 
 For a deeper discussion, see: https://github.com/open-telemetry/opentelemetry-specification/issues/165
 
+## Running Tests Locally
+
+1. Go to your Contrib repo directory. `cd ~/git/opentelemetry-python-contrib`.
```

Comment:
Shouldn't assume or prescribe a location to checkout the git repo at. May be use a placeholder name, relative path or skip `cd` command entirely?